pkg/test/framework/suite_test.go

```diff
@@ -450,6 +450,58 @@ func TestSuite_DoubleInit_Error(t *testing.T) {
 	g.Expect(errCode2).NotTo(Equal(0))
 }
 
+func TestSuite_GetResource(t *testing.T) {
+	defer cleanupRT()
+	g := NewGomegaWithT(t)
```

Comment:
You have to create `g` in each subtest, otherwise it will end up failing the parent test from the subtest.